docs/src/pages/core-concepts/astro-components.md

```diff
@@ -303,7 +303,7 @@ It’s important to note that Astro **won’t** transform HTML references for yo
 
 Since `src/pages/about.astro` will build to `/about/index.html`, you may not have expected that image to live at `/about/thumbnail.png`. So to fix this, choose either of two options:
 
-#### Option 1: Absolute URLs
+### Option 1: Absolute URLs
```

Comment:
same for these, meant to be h4 so that they don't show up in sidebar

Reply:
ah, ok